### PR TITLE
New package: noto-fonts-ttf 20161003

### PR DESCRIPTION
**Pull request by pbui**
This is a package for [Google Noto Fonts](https://www.google.com/get/noto/).  While the google-fonts-ttf includes a version of these fonts, it is not complete (e.g. is missing Noto Sans).  This package is a more complete version of the font family.

### CODE REVIEW

**Comment by dutsky** [comment]
Sorry, i'm neither dev nor important guy here, but why no Sans? I'm just interested because Noto Fonts needed for KDE5

**Comment by pbui** [comment]
Ah, sorry, I meant that the google-fonts-ttf package, which is already in void-packages, includes Noto Sans and Noto Serif, but not Noto Mono.

This new package includes all three and only the Noto family (while the google-fonts-ttf package includes many more fonts).

**Comment by aeosynth** [comment]
there's an unresolved issue (#4436) about whether noto should be packaged. apparently void is weird about packaging fonts.

File: srcpkgs/noto-fonts-ttf/template [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+license="OFL-1.1"
+homepage="https://www.google.com/get/noto/"
+distfiles="https://github.com/googlei18n/noto-fonts/archive/${_githash}.tar.gz"
+checksum=02075aaee1f83c876f0d9ef66563218be9871980e642aaf3beb037bee22005d2
+conflicts="google-fonts-ttf>=0"
+
+do_install() {
+	vmkdir usr/share/fonts/TTF
+	install -m644 hinted/Noto*.ttf ${DESTDIR}/usr/share/fonts/TTF
+	vlicense LICENSE
```
Comment (asergi) [inline srcpkgs/noto-fonts-ttf/template]: `SIL` license is part of `base-files`. So, `vlicense` is unnecessary.

**Comment by pbui** [comment]
@asergi I've updated the template to no longer include `vlicense`.

**Pull request merged**